docs/user/user/server-knobs/pings/product-integration.md

```diff
@@ -1,8 +1,9 @@
 # Product Integration
 
-Glean provides a general Nimbus feature named `glean` that can be used for configuration of pings. Simply select the `glean` feature along with your Nimbus feature from the Experimenter UI when configuring the experiment or rollout (see https://experimenter.info for more information on multi-feature experiments).
+Glean provides a general Nimbus feature named `glean` that can be used for configuration of pings. Simply select the `glean` feature along with your Nimbus feature from the Experimenter UI when configuring the experiment or rollout (see [Nimbus] documentation for more information on multi-feature experiments).
 
-The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the require metric configuration. The format of the configuration is defined in the [Experimenter Configuration](./experimenter-configuration.md) section. If a ping is not included, it will default to the value found in the pings.yaml. Note that this can also serve as an override for Glean builtin pings disabled using the Configuration property `enable_internal_pings=false` during initialization.
+The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the require metric configuration. The format of the configuration is defined in the [Experimenter Configuration] section. If a ping is not included, it will default to the value found in the pings.yaml. Note that this can also serve as an override for Glean builtin pings disabled using the Configuration property `enable_internal_pings=false` during initialization.
```

Comment:
```suggestion
The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the required metric configuration. The format of the configuration is defined in the [Experimenter Configuration] section. If a ping is not included, it will default to the value found in the pings.yaml. Note that this can also serve as an override for Glean builtin pings disabled using the Configuration property `enable_internal_pings=false` during initialization.
```